test/tstLinearBVH.cpp

```diff
@@ -273,7 +273,8 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(buffer_optimization, DeviceType,
                tt::per_element());
   };
 
-  BOOST_CHECK_NO_THROW(bvh.query(ExecutionSpace{}, queries, indices, offset));
+  BOOST_CHECK_NO_THROW(
+      ArborX::query(bvh, ExecutionSpace{}, queries, indices, offset));
```

Comment:
I'd prefer if you did not qualify the call here and below

Reply:
I don't understand why you want this. We are in the tests, not inside the ArborX namespace. Even if ADL does what you want, it is much clearer to me whether this function is from ArborX source or from somewhere in the tests when it's qualified.